tests/integration-v1/cattletest/core/test_allocation.py

```diff
@@ -606,7 +607,8 @@ def test_volumes_from_constraint(new_context):
         c2 = new_context.create_container_no_success(startOnCreate=False,
                                                      dataVolumesFrom=[c1.id])
 
-        c1 = c1.start()
+        c1 = client.wait_success(c1.start())
+        assert c1.state == 'running'
```

Comment:
@ibuildthecloud this needed to change because it is now feasible for c2 to get to allocation before c1 and thus fail one the fact that c1 isn't running.

Does this make sense to you?